jekyll/_docs/build-image-precise.md

```diff
@@ -8,10 +8,6 @@ changefreq: "weekly"
 
 Occasionally, bugs in tests arise because CircleCI's environment differs slightly from your local environment.
 In the future, we plan to allow as much of this to be configurable as possible.
-Please [contact us](mailto:support@circleci.com) if some part of our environment is not suitable for you, and we will try to come up with a workaround.
-
-If any version is not listed here, SSH into our build boxes to check it manually (and [contact us](mailto:support@circleci.com)
```

Comment:
As feature requests and newer versions are handled by discuss our docs should not ask customers to email us for these things.